kirin/gtfs_rt/tasks.py

```diff
@@ -148,28 +158,5 @@ def gtfs_poller(self, config):
             return
 
         logger.debug("polling of %s", config.get("feed_url"))
-
-        # We do a HEAD request at the very beginning of polling and we compare it with the previous one to check if
-        # the gtfs-rt is changed.
-        # If the HEAD request or Redis get/set fail, we just ignore this part and do the polling anyway
-        if not _is_newer(config):
-            new_relic.ignore_transaction()
-            manage_db_no_new(connector_type=ConnectorType.gtfs_rt.value, contributor_id=contributor.id)
-            return
-
-        try:
-            response = _retrieve_gtfsrt(config)
-            response.raise_for_status()
-        except Exception as e:
-            manage_db_error(
-                data="",
-                connector_type=ConnectorType.gtfs_rt.value,
-                contributor_id=contributor.id,
-                error="Http Error",
-                is_reprocess_same_data_allowed=True,
-            )
-            logger.debug(six.text_type(e))
-            return
-
-        wrap_build(KirinModelBuilder(contributor), response.content)
+        _poll_gtfs_rt_feed(config, logger, contributor)
         logger.info("%s for %s is finished", func_name, contributor.id)
```

Comment:
Not your code, but on the way (these logs are heavy when looking for purge finish-logs):
```suggestion
        logger.debug("%s for %s is finished", func_name, contributor.id)
```